code-challenges.js

```diff
@@ -17,33 +17,154 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+
+
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+// create a function boilingPoint
+ // function takes in one argument
+  // arguements named: above, equal, and below
+// create a condition that determines whether the tempurature is above, below, or at boiling point
+ // if the temp is above 212 
+  // return above 
+ // is temp is equal to 212
+  // return equal
+ // else 
+  // return below
+
+const boilingPoint = (temp) => {
+    if (temp > 212){
+        return `${temp} is above boiling point`
+    } else if (temp === 212){
+        return `${temp} is at boiling point`
+    } else {
+        return `${temp} is below boiling point`
+    }
+}
+
+// use function invocation in the console.log to input each tempurature
+
+console.log(boilingPoint(temp1))
+console.log(boilingPoint(temp2))
+console.log(boilingPoint(temp3))
+
+// Outputs:
+// 42 is below boiling point
+// 350 is above boiling point
+// 212 is at boiling point
+
+
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+
+
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+// create a variable numbersCombined
+    // combine the two arrays with the accessor .concat
+// console.log numbersCombined using the informational command .length
+
+let numbersCombined = myNumbers1.concat(myNumbers2)
+
+console.log(numbersCombined.length)
+
+// output: 10
+
+
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 eilrahC"
 
+
+
+
 const currentCohort = "Charlie 2022"
 
+// create a function called reverseCohort
+    // inside the code block start by turning the string into an array .split
+    // reverse the array using mutator .reverse
+    // turn the array back into a string .join
+    // return the last variable 
+// console.log using function invocation
+
+const reverseCohort = (string) => {
+    
+    let arrayCohort = string.split("")
+
+    let reverseArray = arrayCohort.reverse()
+
+    let revStringCohort = reverseArray.join("")
+
+    return revStringCohort
+}
+
+console.log(reverseCohort(currentCohort))
+
+// output: 2202 eilrahC
```

Comment:
Awesome work! 

So it looks like you figured it out Strings are a not a data type that reverse() will work on. 

A lot of our job as developers will be taking in data and changing it's structure while maintaining its meaning. 

so in this example you have successfully: 
1. turned a string into an array
2. transformed that array in some meaningful way
3. turned the array back into a string

A lot of times we come across tools that are very specific to what kind of data they work no and this exercise was meant to push you into experiencing that. Great work!